google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -372,7 +372,7 @@ def read table, columns, keys: nil, index: nil, limit: nil,
         # @yield [commit] The block for mutating the data.
         # @yieldparam [Google::Cloud::Spanner::Commit] commit The Commit object.
         #
-        # @return [Boolean] Returns `true` if the operation succeeded.
+        # @return [Time] The timestamp at which the operation committed.
```

Comment:
I think it's best to use a precise definition, and if it signals that there is a bit more to the story (a transaction is at work), so much the better. But your call.